### PR TITLE
add memory for bot non reg phosphore

### PR DESCRIPTION
**Pull request by henrily-ledger**
### 📝 Description

Add memory because test had memory limit

### ❓ Context

- **Impacted projects**: `` 
- **Linked resource(s)**: `` 

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

**Review by valpinkman — APPROVED** [review]
let's go 🚀

**Review by haammar-ledger — COMMENTED** [review]
Looks neat, but don't we need to do the same for the other seeds' workflows?

**Comment by valpinkman** [comment]
> Looks neat, but don't we need to do the same for the other seeds' workflows?

Depending on the required amount of work..... but it's easy to add it to other jobs indeed

File: .github/workflows/bot-nonreg-phosphore.yml [hunk 1]
```diff
@@ -44,6 +44,8 @@ jobs:
 
   run-bot:
     runs-on: ${{ needs.start-runner.outputs.label }}
+    env:
+      NODE_OPTIONS: "--max-old-space-size=7168"
```
Comment (gre) [inline .github/workflows/bot-nonreg-phosphore.yml:48]: mmh we have 6 non reg bots, shouldn't this be for all?

**Review by hedi-edelbloute — APPROVED** [review]

**Pull request closed without merging**